Src/Common/Domain/implementation/VolunteerOpportunity.cs

```diff
@@ -8,42 +9,60 @@ public class VolunteerOpportunity : IVolunteerOpportunity
         #region Constructors
 
         public VolunteerOpportunity()
-            : this(null)
+            : this(null, null)
         {
         }
 
-        public VolunteerOpportunity(IUser createdBy)
+        public VolunteerOpportunity(IUser createdBy, IOrganization organization)
         {
             CreatedBy = createdBy;
+            Organization = organization;
 
             RecordId = 0;
-            StartTime = DateTime.Today;
-            StopTime = DateTime.Today;
+
+            ProjectName = string.Empty;
+            ProjectDescription = string.Empty;
+
+            StartTime = DateTime.MinValue;
+            StopTime = DateTime.MinValue;
+
+            NumVolunteersNeeded = 0;
+            MaxNumVolunteers = 0;
         }
 
         #endregion
 
         #region Properties
 
         public virtual IUser CreatedBy { get; set; }
+        public virtual IOrganization Organization { get; set; }
+
         public virtual int RecordId { get; set; }
+
+        public virtual string ProjectName { get; set; }
+        public virtual string ProjectDescription { get; set; }
+
         public virtual DateTime StartTime { get; set; }
         public virtual DateTime StopTime { get; set; }
 
+        public virtual int NumVolunteersNeeded { get; set; }
+        public virtual int MaxNumVolunteers { get; set; }
+
         #endregion
 
         #region Public Methods
 
         public virtual bool IsCurrent()
         {
-            return StartTime >= DateTime.Today;
+            return StartTime == DateTime.MinValue || StartTime >= DateTime.Today;
         }
 
         public virtual bool IsPrevious()
         {
-            return StartTime < DateTime.Today;
+            return StartTime < DateTime.Today && StartTime != DateTime.MinValue;
         }
 
+

```

Comment:
I think this is an extra line. You should have one blank line between the } and the #endregion. It's hard to tell in this view.